crates/router/src/core/errors.rs

```diff
@@ -154,7 +154,8 @@ impl From<ConfigError> for ApplicationError {
 
 fn error_response<T: Display>(err: &T) -> actix_web::HttpResponse {
     actix_web::HttpResponse::BadRequest()
-        .append_header(("Via", "Juspay_Router"))
+        .append_header((headers::HSTS, 31536000))
```

Comment:
The numerical value could have been a constant.

I'd prefer if it was explicit as to how the number was obtained:

```rust
const HSTS_MAX_AGE: u32 /* or u64 */ = 60 * 60 * 24 * 365; // 1 year
```

Also, shouldn't the header value be `max-age=31536000` and not just the number?